packages/server-wallet/src/models/channel.ts

```diff
@@ -96,7 +96,7 @@ export class Channel extends Model implements RequiredColumns {
 
   static jsonAttributes = ['vars', 'participants'];
 
-  static async forId(channelId: Bytes32, tx: Transaction | undefined): Promise<Channel> {
+  static async forId(channelId: Bytes32, tx?: Transaction): Promise<Channel> {
```

Comment:
I had hoped that making `tx` a required argument would defend against developers unknowingly querying for data outside the context of a transaction. If you are performing a transaction, and you wrote to channel `'0xf00'`, then `Channel.forId('0xf00', undefined)` would return different data than `Channel.forId('0xf00', ongoingTx)`

Reply:
I see, okay will revert